.rubocop.yml

```diff
@@ -0,0 +1,3 @@
+AllCops:
+  Exclude:
+    - '**/gen-rb/**/*'
```

Comment:
Maybe generated code should also be checked, so that we know compilator generates compliant code